core/src/avm2/globals/xml.rs

```diff
@@ -210,3 +210,21 @@ pub fn call_handler<'gc>(
         .construct(activation, args)?
         .into())
 }
+
+pub fn node_kind<'gc>(
+    _activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let this = this.unwrap();
+    let xml = this.as_xml_object().unwrap();
+    let name = match &*xml.node().kind() {
+        E4XNodeKind::Text(_) => "text",
+        E4XNodeKind::CData(_) => "cdata",
```

Comment:
The AS3 docs don't mention `cdata`, should it be here? If so, can you add a comment mentioning this?

Reply:
good point:
```as
var o: XML = <data><![CDATA[some cdata]]></data>;
trace(o.*[0].nodeKind());
trace(o.*[0]);
```
```
text
some cdata

```

Reply:
Fixed, thanks!